packages/eslint-plugin/rules/jsx-a11y-anchor-has-content.js

```diff
@@ -0,0 +1,161 @@
+/**
+ *
+ * @file Wraps eslint-plugin-jsx-a11y/anchor-has-content to allow usage with createInterpolateElement.
+ *
+ */
+
+/**
+ * External dependencies
+ *
+ * eslint-plugin-jsx-a11y/anchor-has-content: to create a wrapper around it, we need to use it
+ * elementType from jsx-ast-utils: to get the type of the node we are working with.
+ */
+const anchorRule = require( 'eslint-plugin-jsx-a11y' ).rules[
+	'anchor-has-content'
+];
+/**
+ * Internal dependencies
+ */
+const {
+	getTextContentFromNode,
+	getTranslateFunctionName,
+	getTranslateFunctionArgs,
+} = require( '../utils' );
+
+const meta = {
+	type: 'problem',
+	docs: {
+		description:
+			'Checks that anchors are not empty and have an href, relies on eslint-plugin-jsx-a11y/anchor-has-content under the hood. Supports usage of createInterpolateElement.',
+	},
+	fixable: 'code',
+	messages: {
+		anchorHasContent:
+			'Anchors must have content and the content must be accessible by a screen reader. Check the first parameter to createInterpolateElement.',
+		invalidMarkup:
+			'The first parameter to createInterpolateElement does not contain valid markup for an anchor.',
+	},
+	schema: [
+		{
+			type: 'object',
+			properties: {
+				components: {
+					type: 'array',
+					items: {
+						type: 'string',
+					},
+					uniqueItems: true,
+					additionalItems: false,
+				},
+			},
+		},
+	],
+};
+
+/**
+ *
```

Comment:
A description for each function could be useful. Applies to the following functions too.

Reply:
done, thank!